clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/view/EcsServerClusterProvider.java

```diff
@@ -214,6 +237,27 @@ public EcsServerClusterProvider(AccountCredentialsProvider accountCredentialsPro
     return clusterMap;
   }
 
+  private ScalableTarget getScalableTarget(AmazonCredentials credentials, AmazonCredentials.AWSRegion awsRegion, String serviceArn, String clusterName) {
```

Comment:
maybe the method name should be `retrieveServiceScalableTarget` ?

Reply:
Renamed.